_extras/checkout.md

```diff
@@ -38,8 +38,9 @@ To get there, try looking for a link that says "Improve this page". In this curr
 
 ![Screen shot of the header for this page with Improve this page circled](../fig/Improve_this_page.svg){: width="800px" }
 
-Alternatively, you can find links to the GitHub repositories for each lesson next to the links to the rendered lessons on the websites listing Data Carpentry,
-Library Carpentry, or Software Carpentry workshops.
+Alternatively, you can find links to the GitHub repositories for each lesson next to the links to the lessons sites on the websites listing [Data Carpentry](https://datacarpentry.org/lessons/), [Library Carpentry](https://librarycarpentry.org/lessons/), or [Software Carpentry](https://software-carpentry.org/lessons/) lessons:
+
+![Screen shot of Instructor Training GitHub repository in a browser with Issues and Pull Requests tabs circled](../fig/Instructor_Training_Repo_0.png){: width="800px" }
```

Comment:
the alt text and this image to not match, and having both the lesson and repository link does not, to me, help clarify the sentence above.  Can we update the image to either have one circled or have annotation?  Then the alt text (the part inside the `[ ]`) should be edited to match the image.

Reply:
Great I did the pull request right.  I can do the changes you suggest. 

Would this alt text do the job?:
 "Screen shot of Software Carpentries Core Lessons in English list in a browser with links to Sites and Repositories in a square". 

Please suggest any edits to the alt text that you may see fit.

I'm not sure about your suggestions to the figure:
- do you prefer  a circle or oval instead of the square? --- I can change the colour too.
- by "update image to have  annotation", do you mean painting the image with some text or arrows or something else?
I can do both or something else. Just let me know.

Sorry for asking too much. This is my first ever pull request and I'm not totally familiar with some terms and the process. 
 
Thanks.

Reply:
that alt text is perfect.  I was thinkign the image might be more helpful if *only* the repository links were highlighted? Since that is what we are trying to help people find?